clusterloader2/api/extensions.go

```diff
@@ -55,3 +55,13 @@ func (d *Duration) ToTimeDuration() time.Duration {
 func (s *Step) IsMeasurement() bool {
 	return len(s.Measurements) > 0
 }
+
+// IsPhase returns true whether a step is a phase-step.
+func (s *Step) IsPhase() bool {
+	return len(s.Phases) > 0
+}
+
+// IsModule returns true whether a step is a module.
+func (s *Step) IsModule() bool {
+	return len(s.Module.Steps) > 0 || s.Module.Path != ""
```

Comment:
I'm slightly baffled by this condition. Why not simply `s.Module.Path != ""`?

In addition, this contradicts the logic included in the proposed `loadModule` function where we currently run a following `if` statement:
```
if len(module.Steps) > 0 {
	return nil, fmt.Errorf("module %v has both path and steps set", module)
}
```

Reply:
Obsolete.